modules/appeals_api/app/controllers/concerns/appeals_api/header_modification.rb

```diff
@@ -7,10 +7,12 @@ module HeaderModification
     included do
       V1_DEV_DOCS = 'https://developer.va.gov/explore/appeals/docs/decision_reviews?version=1.0.0'
       V2_DEV_DOCS = 'https://developer.va.gov/explore/appeals/docs/decision_reviews?version=2.0.0'
+      RELEASE_NOTES_LINK = 'https://developer.va.gov/release-notes/appeals'
```

Comment:
I think it'd be useful to link to the specific release that deprecated an API version, if possible. Since the reasoning for deprecation will change between releases, maybe we should have the release link be a method on the controller similar to `sunset_date`. What do you think?

Reply:
There are no anchor tags for specific release notes. The appeals release notes are on a single page.

Reply:
Browsers have supported using any DOM's `id` attribute as an allowable fragID in the url, and will behave the same as (and actually take precedence over) an anchor tag.

So we can link to a specific release by linking to e.g. `https://developer.va.gov/release-notes/appeals#january-31-2019`

Reply:
We'll have to update this when we have official release notes. As of this moment the release notes page is relatively small, so it's a pretty negligible difference, but I think it's a good idea!